src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Windows_NT.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -20,6 +20,7 @@
     <NativeBinary Include="$(BinDir)mscorrc.dll" />
     <NativeBinary Include="$(BinDir)sos.dll" />
     <NativeBinary Include="$(BinDir)System.Private.CoreLib.ni.dll" />
```

Comment:
Since there are not debug binaries (which introduce additional dependencies that Release CRT build cannot support) for CRT that can be distributed, we statically link it for Debug build.